packages/flutter/lib/src/material/search_anchor.dart

```diff
@@ -657,6 +665,7 @@ class _SearchViewRoute extends PopupRoute<_SearchViewRoute> {
     if (willDisposeSearchController) {
       searchController.dispose();
     }
+    viewOnClose?.call();
```

Comment:
Should we call this in `didPop`?

Reply:
@QuncCccccc do you have any suggestion of another place to put this?

Reply:
I've chosen to put it there because is when we can be sure nothing related to the view is presented anymore

Reply:
I see. Then I think moving this call to didPop() will also do the same thing:)?